python/lsst/meas/extensions/scarlet/io.py

```diff
@@ -648,6 +654,81 @@ def __init__(self, psfs, model_psf, bbox, dtype):
         )
 
 
+def multibandDataToScarlet(
+    modelData,
+    blendId,
+    observedPsfs=None,
+    dtype=np.float32,
+    mExposure=None,
+    footprint=None,
+):
+    """Convert the store data model into a scarlet lite blend,
+    including observation information.
+
+    While the typical use case in the science pipelines is to attach
+    scarlet models as footprints to a `SourceCatalog`, it can be advantageous
+    to load an entire multi-band blend. This requires (at a minimum) the
+    PSF in each observed band for the final model.
+
+    Parameters
+    ----------
+    modelData : `ScarletModelData`
+        The model for all of the blends in a given tract/patch.
+    blendId : `int`
+        The source record ID of the parent record in the catalog.
+    observedPsfs : `list` of `lsst.detection.Psf`
+        The PSF for each observed image.
+        Typically this is obtained using
+        ```butler.get("deep_Coadd_calexp.psf", **dataId)``` for a given
+        (tract, patch, band).
+        If `mExposure` is not `None` then the observed PSFs are generated
+        automatically, otherwise this parameter is required.
+    dtype : `numpy.dtype`
+        Datatype for the rendered model. If `mExposure` is not `None` then
+        this parameter is ignored and the `dtype` of the image is used.
+    mExposure : `lsst.afw.image.MultibandExposure`
+        The observed exposure in each band.
+        This is not required in order to render the models into numpy arrays,
+        however it is required if the user plans to perform a warm restart
+        using the stored models.
+    footprint : `lsst.afw.detection.Footprint`
+        The footprint of the parent blend.
+        This is only required if the user desires to perform a warm restart
+        and wants to mask out the pixels outside of the parent footprint
+        similar to when scarlet was executed in the science pipelines.
+
+    Returns
+    -------
+    blend : `scarlet.lite.LiteBlend`
+        The full scarlet model for the blend.
+    """
+    # Import here to prevent circular import
```

Comment:
As above, surely you can refactor these modules to avoid potential circular imports?

Reply:
In general I agree and most of the refactoring in this ticket was to clean things like this up. However in this case I think that it would add an unnecessary module containing only a single function.